hype-examples/commons/src/main/scala/com/spotify/hype/examples/GSUtilCp.scala

```diff
@@ -0,0 +1,21 @@
+package com.spotify.hype.examples
+
+import org.slf4j.LoggerFactory
+
+import scala.sys.process._
+
+case class GSUtilCp(inPath: String,
+                    outPath: String) extends HypeModule[Int] {
+
+  @transient private lazy val log = LoggerFactory.getLogger(classOf[GSUtilCp])
+
+
+  override def getFn = {
+    s"gcloud auth activate-service-account --key-file=${System.getenv().get("GOOGLE_APPLICATION_CREDENTIALS")}" !
+    val cmd = s"gsutil -m cp -r $inPath $outPath"
+    log.info(cmd)
+    cmd !
+  }
+
+  override def getImage: String = "us.gcr.io/datawhere-test/hype-examples-base:4"
```

Comment:
I believe this is a private repository. Since this is a public example, it would be better to use the public spotify repository for images we use in our public examples.